src/generator/AutoRest.Go/rakefile.rb

```diff
@@ -9,17 +9,18 @@
 AUTOREST     = "../../core/AutoRest/bin/Debug/net451/win7-x64/AutoRest.exe"
 
 SWAGGER_VERSIONS = {
+	# asazure: {version: "2016-05-06"},
```

Comment:
I could regen, yes. I updated the rake file just to keep it updated, and add comments on what we are missing.